src/jdk.incubator.jextract/share/classes/jdk/internal/clang/LibClang.java

```diff
@@ -31,12 +31,12 @@
 
 public class LibClang {
     private static final boolean DEBUG = Boolean.getBoolean("libclang.debug");
-    private static final boolean NO_CRASH_RECOVERY = Boolean.getBoolean("libclang.disable_crash_recovery");
+    private static final boolean CRASH_RECOVERY = Boolean.getBoolean("libclang.crash_recovery");
 
     public static Index createIndex(boolean local) {
         Index index = new Index(Index_h.clang_createIndex(local ? 1 : 0, 0));
-        Index_h.clang_toggleCrashRecovery(NO_CRASH_RECOVERY ? 0 : 1);
-        if (DEBUG && NO_CRASH_RECOVERY) {
+        Index_h.clang_toggleCrashRecovery(CRASH_RECOVERY ? 1 : 0);
+        if (DEBUG && !CRASH_RECOVERY) {
```

Comment:
Crash recovery is disabled by default, right? Shouldn't the debug for enabling? (because that is the explicit config behaviour - or else this message will be printed always)

Reply:
This was the way it was before when crash recovery was always being disabled. It would always print the message when running in debug mode.

I guess now we can print whether the crash recovery is enabled or disabled instead.

Reply:
Looks good. Go ahead!